src/main.cpp

```diff
@@ -6719,7 +6719,7 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
     else if (strCommand == NetMsgType::XBLOCKTX && !fImporting && !fReindex && !IsInitialBlockDownload() &&
              IsThinBlocksEnabled())
     {
-        return CXThinBlockTx::HandleMessage(vRecv, pfrom);
+        return CXThinBlockTx::HandleMessage(vRecv, pfrom, strCommand);
```

Comment:
With a view to message dispatch being just a function of vRecv and pfrom I'd like to drop the strCommand if possible (propogating the constant).  It would probably reduce files touched by this PR from 5 to 3 as well.